### PR TITLE
Support print parse error to log

### LINKED ISSUE #1515: LOG detailed when GQLParser failure
`E1225 10:03:01.519412 39245 ExecutionPlan.cpp:23] SyntaxError: syntax error at 1.19512`

Here, specific statements should be output in the log, which is convenient for troubleshooting when batch processing.

### PR DESCRIPTION
**Pull request by laura-ding**
close #1515

### CODE REVIEW

**Comment by laura-ding** [comment]
jenkins go

File: src/console/CmdProcessor.cpp [hunk 1]
```diff
@@ -431,29 +431,8 @@ void CmdProcessor::processServerCmd(folly::StringPiece cmd) {
...
-            verbose = headMsg + std::string(&cmd[start-1], end - start + 1) + "'";
-        } else if (std::regex_search(*msg, result, single)) {
-            auto start = folly::to<size_t>(result[1].str());
-            auto end = start + 8;
-            end = end > cmd.size() ? cmd.size() : end;
-            verbose = headMsg + std::string(&cmd[start-1], end - start + 1) + "'";
-        }
         std::cout << "[ERROR (" << static_cast<int32_t>(res)
-                  << ")]: " << verbose << "\n";
+                  << ")]: " << *resp.get_error_msg() << "\n";
```
Comment (Shylock-Hg) [inline src/console/CmdProcessor.cpp]: (resp.get_error_msg == nullptr ? "" : *resp.get_error_msg() ) better ?

File: src/common/base/Status.h [hunk 2]
```diff
@@ -81,7 +81,8 @@ class Status final {
         return Status(k##ERROR, msg);                   \
     }                                                   \
                                                         \
-    static Status ERROR(const char *fmt, ...) {         \
+    static Status ERROR(const char *fmt, ...)           \
+        __attribute__((format(printf, 1, 2))) {         \
```
Comment (Shylock-Hg) [inline src/common/base/Status.h]: Consider `FOLLY_PRINTF_FORMAT_ATTR`?

Comment (laura-ding) [reply src/common/base/Status.h]: Thanks, I'm deliberately not relying on folly

File: src/console/CmdProcessor.cpp (hunk 1 above)
Comment (laura-ding) [reply src/console/CmdProcessor.cpp]: yeah

**Pull request merged**